src/game/mod.rs

```diff
@@ -141,6 +142,10 @@ impl Game {
         self.visit(Location::home()).unwrap_or_default();
     }
 
+    pub fn add_gold(&mut self, amount: i32) {
```

Comment:
This function doesn't seem to add much, i'd rather just do `self.gold += amount;` directly (which is what's being done elsewhere to add gold). Also it doesn't need to be public.

Reply:
Followed suggestion :wink: